src/environment.ts

```diff
@@ -105,16 +75,6 @@ export class SecureEnvironment implements MembraneBroker {
         };
         this.getRedValue = redEnvFactory(this, blueHooks);
         this.getBlueValue = blueProxyFactory(this);
-        // remapping intrinsics that are realm's agnostic
```

Comment:
the env instance doesn't do remaps anymore, those are done right after by the caller of this constructor.